### DISCUSSION TITLE
divide utilities not transition-able

### DISCUSSION THREAD

**Discussion by joulev** [tailwindlabs/tailwindcss#7940] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.0.23

**What build tool (or framework if it abstracts the build tool) are you using?**

N/A (reproduced in Tailwind Play, see below)

**What version of Node.js are you using?**

N/A

**What browser are you using?**

Firefox

**What operating system are you using?**

Kubuntu (Linux)

**Reproduction URL**

https://play.tailwindcss.com/wF4QIsLSDs

When hover over the flex, the divider between the two children changes the colour immediately, while they should follow the transition duration like the main element borders instead.

**Describe your issue**

Since `divide` doesn't style the current element but some element related to it (in this case, all children except the last), `transition` doesn't really work. Not even sure if this is fixable (I can't think of any sensible way to modify the CSS for this on the spot), but yes, I'd like to animate the `divide` borders, be it by putting `transition` on the main element or all children elements.

**Comment by adamwathan** [comment, marked as answer]
Hey! As far as I can tell this works, you just need to put the transition/duration on the child elements:

https://play.tailwindcss.com/112vcQwoYC

**Reply by joulev** [reply]
Oh, I also put the `transition` to the children in my Tailwind Play but forgot to add the corresponding `duration`s… 🤦‍♂️ 

Thanks for the help!
